Avoid warning when no PREMIS results, refs #13348

djjuhasz opened the pull request:
Prevent a PHP Warning when adding an information object to ES that has no
related PREMIS metadata.

mcantelon committed:
Add accession events, refs #13315

This adds support for accession events, based on the Canadian Archival
Accession Information Standard (CAAIS).

* Add new AtoM object type: accession events
* Add accession event type taxonomy
* Add "Physical transfer" accession event type term
* Add method to accession model to get an accession's accession events
* Add ability to add/edit accession events via the web UI
* Add Elasticsearch document type for accession events
* Update accession search/indexing to incorporate accession event
  types, dates, agents, and notes

mcantelon committed:
CSV accession import event support, refs #13315

Adds functionality, to CSV accession import, to allow accession events
to be imported.

mcantelon committed:
Warn of duplicate actor identifiers, refs #13373

Added, in the web UI and during authority record CSV import/updating,
warnings when an authority record identifier has already been used.

djjuhasz committed:
Avoid warning when no PREMIS results, refs #13348

Prevent a PHP Warning when adding an information object to ES that has no
related PREMIS metadata.

sbreker reviewed (approved):
Looks great! Thanks @djjuhasz!

mcantelon committed:
Fix multi-row JS/date picker issue, refs #13315

There were issues with the date picker being used as a field with the
multi-row JS. The date picker generates table HTML that was interfering
with the multi-row JS and when the multi-row JS cloned rows the date
picker in the new row wasn't being activated. Fixed issues.
